ci-operator/config/openshift-service-mesh/sail-operator/openshift-service-mesh-sail-operator-release-3.0__ocp-4.14.yaml

```diff
@@ -0,0 +1,135 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: upstream-1.24
+  use_build_cache: true
+releases:
+  latest:
+    release:
+      architecture: multi
+      channel: stable
+      version: "4.14"
```

Comment:
Does 4.14 version still relevant for us to test on?

Reply:
Yes, 3.0.z is supported on OCP 4.14 -- 4.18